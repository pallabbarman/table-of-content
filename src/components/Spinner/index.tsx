const Spinner = () => {
    return (
        <div className="h-20 w-20 animate-spin rounded-full border-8 border-gray border-t-sky" />
    );
};

export default Spinner;
