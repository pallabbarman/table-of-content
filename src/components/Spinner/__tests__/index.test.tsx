import { renderAppToSnapshot } from 'test_utils/render';
import Spinner from '..';

describe('Spinner', () => {
    test('should render the default component', () => {
        renderAppToSnapshot(<Spinner />);
    });
});
