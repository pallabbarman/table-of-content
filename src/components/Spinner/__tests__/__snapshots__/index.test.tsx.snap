// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Spinner > should render the default component 1`] = `
<body>
  <div>
    <div
      class="h-20 w-20 animate-spin rounded-full border-8 border-gray border-t-sky"
    />
  </div>
</body>
`;
